.github/workflows/ci.yml

```diff
@@ -9,47 +11,57 @@ on:
       - main
 
 jobs:
+  # Check whether workflow is skippable
   pre_check:
     name: Pre-Check
     runs-on: ubuntu-latest
     outputs:
       # Returns 'true' if workflow is skippable for one of the following reasons:
       # - Exact same files have been successfully checked in older run
-      # - Only files that do not require checks have been changed (paths_ignore)
+      # - Only files that do not require checks have been changed (paths_ignore) and older commits were successful
       should_skip: ${{ steps.skip_check.outputs.should_skip }}
+      skipped_by: ${{ steps.skip_check.outputs.skipped_by }}
     steps:
       - name: Check if workflow is skippable
         id: skip_check
-        uses: fkirc/skip-duplicate-actions@master
+        # Switch back to fkirc/skip-duplicate-actions once changes have been merged
```

Comment:
What changes?

Reply:
We can switch back once the following pull requests have been merged: 
* https://github.com/fkirc/skip-duplicate-actions/pull/152
* https://github.com/fkirc/skip-duplicate-actions/pull/153
I'm going to submit a pull request as soon as this is the case.

Those changes gives us access to more information about the skip-duplicate-actions, for example to prevent skipping the end-to-end tests in some cases.